### PR TITLE
Develop emily

### PR DESCRIPTION
**Pull request by kathrynline**

### CODE REVIEW

File: detect_analysis.R [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+author: "Emily Linebarger"
+date: "2/17/2021"
+output: pdf_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+library(data.table)
+
+dt = fread("C:/Users/eklin/Documents/repos/data_557/prepped_data/sixmonth.csv")
```
Comment (agale123) [inline detect_analysis.R]: To make this easier for others to reproduce, can this be made into a relative path?

File: detect_analysis.R [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+```
+
+First, run a simple scatter to see what the correlation between the overall score and the "early detection" sub-component looks like. 
+```{r exploratory_analysis}
```
Comment (agale123) [inline detect_analysis.R]: Is this an Rmd file or an R file? It looks like Rmd, but the extension is R.

File: recalculate_weights.R [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+dt[, early_detection_new:=early_detection*early_detection_wt]
+dt[, rapid_response_new:=rapid_response*rapid_response_wt]
+dt[, robust_hs_new:=robust_health_sector*robust_hs_wt]
+dt[, commitments_new:=commitments*commitments_wt]
+dt[, risk_environment_new:=risk_environment*risk_environment_wt]
+
+dt[, overall_new:=prev_emergence_new + early_detection_new + rapid_response_new + robust_hs_new + commitments_new + risk_environment_new]
+dt[, overall_new:=round(overall_new, digits=1)]
+
+# Can get pretty close! Just a little bit off.
```
Comment (agale123) [inline recalculate_weights.R:32]: This is great that you were able to validate those weights!

**Review by agale123 — APPROVED** [review]

File: detect_analysis.R (hunk 1 above)
Comment (kathrynline) [reply detect_analysis.R]: Yep! Good call.

File: detect_analysis.R (hunk 2 above)
Comment (kathrynline) [reply detect_analysis.R]: This one was a mistake - I'll delete.

**Pull request merged**